src/Microsoft.Health.Dicom.Core/Features/Validation/ElementRequiredLengthValidation.cs

```diff
@@ -72,10 +77,15 @@ private static bool TryGetAsString(DicomElement dicomElement, out string value)
 
         private void ValidateStringLength(DicomVR dicomVR, string name, string value)
         {
-            value = value ?? "";
+            value ??= "";
```

Comment:
the 'ol null-coalescing-conditional-assignment-operator, I see!

Reply:
A classic!